### PR TITLE
Cl userderma net #188

### PR DESCRIPTION
**Pull request by samuelWilliams99**
Wire cl_userderma to net messages, card #188

### CODE REVIEW

**Review by sarahsturgeon — CHANGES_REQUESTED** [review]
We'll need to wait until V's big PR is merged, then deal with conflicts here, before merging

File: lua/cfcfactions/core/cl/dermas/cl_usersderma.lua
Comment (justinmuskopf) [inline lua/cfcfactions/core/cl/dermas/cl_usersderma.lua]: `local factioneers = {}`

Comment (justinmuskopf) [reply lua/cfcfactions/core/cl/dermas/cl_usersderma.lua]: (And propagate to the rest of the file, of course)

File: lua/cfcfactions/core/cl/dermas/cl_usersderma.lua
Comment (justinmuskopf) [inline lua/cfcfactions/core/cl/dermas/cl_usersderma.lua]:
```suggestion
local ViewUserDerma = {}
```
Or something like that?
@brandonsturgeon ideas?

**Review by justinmuskopf — CHANGES_REQUESTED** [review]
I need to revisit this, brb

File: lua/cfcfactions/core/cl/dermas/cl_usersderma.lua
Comment (samuelWilliams99) [reply lua/cfcfactions/core/cl/dermas/cl_usersderma.lua]: I've changed this for cl_usersderma and sv_users, not sure if you wanted me to also do sv_users, please take a look.

**Comment by sarahsturgeon** [comment]
Tested on CFC Test server with no errors and expected print output

File: lua/cfcfactions/core/cl/dermas/cl_usersderma.lua
Comment (ghost) [inline lua/cfcfactions/core/cl/dermas/cl_usersderma.lua:6]: This needs to be undone (for now) since its used to register it to the menubar. Its messy but it works, for now.

**Review by ghost — CHANGES_REQUESTED** [review]
Minor changes

File: lua/cfcfactions/core/cl/dermas/cl_usersderma.lua
Comment (samuelWilliams99) [reply lua/cfcfactions/core/cl/dermas/cl_usersderma.lua:6]: You want to keep `cfcFactions:RegisterDermaMenu( "View Users", Panel, 1 )`?

Comment (ghost) [reply lua/cfcfactions/core/cl/dermas/cl_usersderma.lua:6]: Yes, for now. Part of my work is redoing that to its no longer needed. Its stupidly messy and overly complex

**Comment by samuelWilliams99** [comment]
Is this finally ready for merge?

**Comment by ghost** [comment]
> 
> 
> Is this finally ready for merge?

Was this tested? It may not be testable till we incorporate [Bleck's UI](https://github.com/brandonsturgeon/cfc_factions_3/pull/70). Not that big of a deal + we can refine it as we go.

**Comment by samuelWilliams99** [comment]
> > Is this finally ready for merge?
> 
> Was this tested? It may not be testable till we incorporate [Bleck's UI](https://github.com/brandonsturgeon/cfc_factions_3/pull/70). Not that big of a deal + we can refine it as we go.

We ran it on a test server and printed the table on client when it was received, all seemed fine

**Comment by ghost** [comment]
All good then

**Pull request merged**
